src/version.h

```diff
@@ -0,0 +1,34 @@
+// Copyright (c) 2017, The Monero Project
```

Comment:
I think we can lose the comment header here too. The original version.h didn't have any either. Not like a few lines of variable declarations constitute copyrightable work, anyway.

Reply:
ok